src/swagger.ts

```diff
@@ -39,6 +39,8 @@ export const SwaggerDocs = (app: INestApplication) => {
   );
 
   const options = {
+    jsonDocumentUrl: 'openapi/json',
+    yamlDocumentUrl: 'openapi/yaml',
```

Comment:
I've just thought: we need to update the test in `/test/docs/get-docs-yaml.api-test.ts` because it tries to read the spec from this endpoint

Reply:
Good spot, have updated.